src/components/Folder/Folder.tsx

```diff
@@ -524,7 +524,7 @@ const Folder = <T extends Conversation | Prompt | DialFile>({
               className={classNames(
                 'relative max-h-5 flex-1 truncate break-all text-left group-hover/button:pr-5',
                 highlightedFolders?.includes(currentFolder.id)
-                  ? 'text-accent-secondary'
+                  ? 'text-accent-primary'
```

Comment:
Should we fix e2e-test also?
https://github.com/epam/ai-dial-chat/pull/518/files#diff-a4cd434c15c2bc55c4a3d976dc1ea82fddbaa185faca40a737e17aa85c83f175